### ISSUE TITLE
(in)equality with LOG(M_E)

### ISSUE THREAD

**Issue by curioustorvald** [minjaesong/terran-basic-lua#2]
PRINT LOG(M_E) == 1.0
    PRINT LOG(M_E) <> 1.0

these two statements both print FALSE, tested with Lua 5.1, 5.2 and 5.3

**Comment by curioustorvald** [comment]
also happens with ```TAN(.25 * M_PI)```

**Comment by curioustorvald** [comment]
Seemingly fixed with the lastest commit. Now it works the same as its underlying Lua implementation. Examples:

Lua:

```
Lua 5.2.4  Copyright (C) 1994-2015 Lua.org, PUC-Rio
> e = 2.7182818284591
> print(math.log(e) == 1.0)
false
> print(math.log(e) ~= 1.0)
true
```

TBASIC:
```

    **** TERRAN BASIC V0.4 ****    

READY.
PRINT LOG(M_E) == 1.0
 FALSE

READY.
PRINT LOG(M_E) <> 1.0
 TRUE

READY.

```

**Comment by curioustorvald** [comment]
Commit 0e0f39610cc35e33e50c9add502345755ae74dcc finally fixed ```LOG(M_E) == 1.0``` being FALSE; it's TRUE now.
